apps/docs/content/guides/self-hosting/docker.mdx

```diff
@@ -115,6 +138,14 @@ psql -h 127.0.0.1 -p 5432 -d postgres -U postgres
 
 The default password is `your-super-secret-and-long-postgres-password`. You should change this as soon as possible using the [instructions below](#update-secrets). By default the database is not accessible from outside the local machine. You can [change this](#exposing-your-postgres-database) by updating the `docker-compose.yml` file.
 
+You will also likely want to connect to your postgres database via an ORM or another direct method other than `psql`
+
+For this you will use the following URL which uses connection pooling by default.
```

Comment:
```suggestion
For this you can use the standard Postgres connection string.
```

As far as I understand, this is the standard connection string and you won't get connection pooling with it. Any reason to believe otherwise?